.circleci/config.yml

```diff
@@ -89,7 +98,9 @@ jobs:
             make ci:diff | while read DIR; do
               cd ${CIRCLECI_WORKSPACE}/${DIR}
               if find . -type f -name "Dockerfile${EXTENSION}" | grep Dockerfile ; then
-                make push;
+                if [[ "flyway" == $DIR ]] || [[ "flyway-mysql57" == $DIR ]]; then
```

Comment:
This means the other images will not be pushed, we can push just `flyway` and `flyway-mysql57`. Why do we need this condition?

Reply:
Thank you, I will remove it at the end as it is under development.

Sometimes there is an automatic update to the master branch and the image is updated...